src/components/fields/index.ts

```diff
@@ -4,8 +4,9 @@ export * from "./chips";
 export * from "./contact-field";
 export * from "./date-field";
 export * from "./date-range-field";
-export * from "./file-upload";
+export * from "./hidden-field";
 export * from "./histogram-slider";
+export * from "./file-upload";
```

Comment:
was moving `file-upload` intended? as it was already in the correct alphabetical order

Reply:
whoops not intended, I have fixed it